src/Workspaces/Core/Portable/Classification/EmbeddedLanguages/ExportEmbeddedLanguageClassifier.cs

```diff
@@ -12,16 +14,55 @@ namespace Microsoft.CodeAnalysis.Classification
     /// </summary>
     [MetadataAttribute]
     [AttributeUsage(AttributeTargets.Class)]
-    internal sealed class ExportEmbeddedLanguageClassifier : ExportAttribute
+    internal class ExportEmbeddedLanguageClassifierAttribute : ExportAttribute
```

Comment:
this type will become public to allow a 3rd party to export their own classifier for their own language.